packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentF24Item.test.tsx

```diff
@@ -9,7 +9,9 @@ describe('NotificationPaymentF24Item Component', () => {
 
   it('renders NotificationPaymentPagoPAItem - should show title of f24Item', () => {
```

Comment:
Add an expect or a separeted test, where we pass a mock function as handleDownloadAttachment and simulate click.
Test that the function i called one times and that is called with expected parameters